CONTRIBUTING.md

```diff
@@ -30,9 +30,48 @@ Task Force Participants grant to the W3C a perpetual, nonexclusive, royalty-free
 
 Nothing in this agreement restricts the Task Force Participant from using their individual contributions as they wish, even if those have later been amalgamated into joint works. Where W3C releases materials under a permissive license such as the W3C Software License or CC-BY, nothing in this agreement should be read to restrict the Task Force Participant from exercising the permissions granted by that license. The Task Force Participant represents that they are legally entitled to grant the above license. If their employer(s) have rights to intellectual property that the Task Force Participant creates that includes the contributions, they represent that they have received permission to make contributions on behalf of that employer or that the employer has waived such rights for the contributions to W3C.
 
-
 ## Decency
 
 The Task Force Participant will participate in the W3C Group in a decent way. Task Force Participants will refrain from defaming, harassing or otherwise offending other participants. The [Section 3.1 of the Process Document](https://www.w3.org/2015/Process-20150901/#ParticipationCriteria) applies, as does the W3C [Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
 
 The Task Force Participant will refrain from sending unsolicited commercial messages to W3C mailing-lists and other promotional activities for personal matters or for third parties. This is especially required from Task Force Participants sending messages to public W3C Groups.
+
+# Chinese translation (中文版)
+
+注：本译文不是官方文档，仅供参考。
+
+## 关于贡献
+
+对此仓库的贡献旨在成为使用[软件和文档许可](http://www.w3.org/Consortium/Legal/copyright-software)的国际化兴趣组和国际化工作组文档的一部分。通过在此承诺，您同意您的贡献许可。
+
+如果您不是该贡献（pull request）的唯一贡献者，请在 pull request 的评论中列出所有贡献者。
+
+要添加一个除了您之外的贡献者，请用如下格式每行列出一个贡献者的 GitHub 用户名：
+
+```
++@github_username
+```
+
+如果您错误地添加了贡献者，可以在评论中删除他们：
+
+```
+-@github_username
+```
+
+如果您代表其他人提出 pull request，但您本人没有参与设计该特性（或编辑该文档），则可以使用上述语法移除自己。
+
+## 关于版权
+
+版权是标准化活动中非常重要的一部分，能够允许标准化组织维持规范的厂商中立性，从而保护工作组内的共识。
+
+在 W3C 小组中开发文档的过程中，任务团（task force）的成员将做出贡献。这些贡献将被整合入联合开发的工作，从而为任务团成员的贡献创造共享版权。大多数 W3C 规范都包含一个致谢章节。
+
+任务团成员对其贡献的版权授予 W3C 永久、非排他性、免版税的全球权利和许可，以 W3C 选择的许可证复制、发布和分发其贡献。此外，任务团成员授予 W3C 版权和许可的作品、基于或包含该作品的全部或部分内容，以及由 W3C 编辑的该作品的任何衍生作品应完全由 W3C 拥有。此外，任务团成员理解 W3C 将能够行使作为任务团成员贡献的版权所有者的所有权利，包括在不通知任务团成员的情况下对侵权者采取法律措施。
+
+本协议中没有任何内容限制任务团成员按照自己的意愿使用他们的个人贡献，即使这些贡献后来被并入联合作品中。如果 W3C 根据一个宽松的许可协议（如 W3C 软件许可证或 CC-BY）发布文档，则本协议中的任何内容不应被理解为会限制任务团成员行使该许可证授予的权利。任务团成员声明他们在法律上有权授予上述许可。如果任务团成员的雇主拥有的知识产权包含任务团成员的贡献，则任务团成员声明他们已经获得代表该雇主作出贡献的许可，或者雇主已为向 W3C 作出贡献放弃了这部分知识产权。
+
+## 得体
+
+任务团成员须以得体的方式参与 W3C 小组，避免诽谤、骚扰或以其他方式冒犯其他参与者。参与任务团的工作适用 [W3C 流程文档第 3.1 节](https://www.w3.org/2015/Process-20150901/#ParticipationCriteria)与 [W3C 职业道德与行为守则](https://www.w3.org/Consortium/cepc/)。
```

Comment:
There is an extra `U+001C` character on this line and a `U+0008` on line 77.

Reply:
Thanks. Should be fixed now.